database.sql

```diff
@@ -1,63 +1,43 @@
--- phpMyAdmin SQL Dump
--- version 4.5.4.1deb2ubuntu2
--- http://www.phpmyadmin.net
---
--- Client :  localhost
--- Généré le :  Jeu 26 Octobre 2017 à 13:53
--- Version du serveur :  5.7.19-0ubuntu0.16.04.1
--- Version de PHP :  7.0.22-0ubuntu0.16.04.1
-
-SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
-SET time_zone = "+00:00";
-
-
-/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
-/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
-/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
-/*!40101 SET NAMES utf8mb4 */;
-
---
--- Base de données :  `simple-mvc`
---
-
--- --------------------------------------------------------
-
---
--- Structure de la table `item`
---
-
-CREATE TABLE `item` (
-  `id` int(11) UNSIGNED NOT NULL,
-  `title` varchar(255) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
-
---
--- Contenu de la table `item`
---
-
-INSERT INTO `item` (`id`, `title`) VALUES
-(1, 'Stuff'),
-(2, 'Doodads');
-
---
--- Index pour les tables exportées
---
-
---
--- Index pour la table `item`
---
-ALTER TABLE `item`
-  ADD PRIMARY KEY (`id`);
-
---
--- AUTO_INCREMENT pour les tables exportées
---
-
---
--- AUTO_INCREMENT pour la table `item`
---
-ALTER TABLE `item`
-  MODIFY `id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
-/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+CREATE TABLE user (
+  id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  pseudo VARCHAR(20) NOT NULL,
+  adress VARCHAR(80) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  photo LONGBLOB NOT NULL,
+  rating INT DEFAULT 5
+);
+
+
+CREATE TABLE product (
+  id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  title VARCHAR(20) NOT NULL,
+  price INT NOT NULL,
+  description TEXT NOT NULL,
+  photo LONGBLOB NOT NULL,
+  status VARCHAR(50) DEFAULT 'en vente',
+  material VARCHAR(50),
+  category_item VARCHAR(50),
+  category_room VARCHAR(50),
+  color VARCHAR(50),
+  `date` DATETIME DEFAULT NOW(),
+  `condition` VARCHAR(20),
+  user_id INT NOT NULL,
+  CONSTRAINT fk_product_user
+        FOREIGN KEY (user_id)
+        REFERENCES user(id)
+);
+
+
+CREATE TABLE cart (
+  id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  status_validation BOOL DEFAULT false,
+  `date` DATETIME DEFAULT NOW(),
+  user_id INT NOT NULL,
+  CONSTRAINT fk_cart_user
+        FOREIGN KEY (user_id)
+        REFERENCES user(id),
+  product_id INT NOT NULL,
+  CONSTRAINT fk_cart_product
+        FOREIGN KEY (product_id)
+        REFERENCES product(id)
+);
```

Comment:
Don't forget to add the On DELETE and ON UPDATE options in order to delete all necessaries data when tables have relations